js/map.js

```diff
@@ -73,45 +71,62 @@ function getTooltip(props){
       <thead>
         <tr>
           <th>Category</th>
-          <th><span class='float-end'>kW</span></th>
-          <th><span class='float-end'>Projects</span></th>
+          <th><span class='float-end'>Energized</span></th>
+          <th><span class='float-end'>#</span></th>
+          <th><span class='float-end'>Planned</span></th>
+          <th><span class='float-end'>#</span></th>
         </tr>
       </thead>
       <tbody>
         <tr>
           <td>Total</td>
           <td><span class='float-end'>${props.total_kw.toLocaleString()}kW</span></td>
           <td><span class='float-end'>${props.total_count.toLocaleString()}</span></td>
+          <td><span class='float-end'>${props.planned_total_kw.toLocaleString()}kW</span></td>
+          <td><span class='float-end'>${props.planned_total_count.toLocaleString()}</span></td>
         </tr>
         <tr>
           <td>Utility</td>
           <td><span class='float-end'>${props.utility_kw.toLocaleString()}kW</span></td>
           <td><span class='float-end'>${props.utility_count.toLocaleString()}</span></td>
+          <td><span class='float-end'>${props.planned_utility_kw.toLocaleString()}kW</span></td>
+          <td><span class='float-end'>${props.planned_utility_count.toLocaleString()}</span></td>
         </tr>
         <tr>
           <td>Community Solar</td>
           <td><span class='float-end'>${props.cs_kw.toLocaleString()}kW</span></td>
           <td><span class='float-end'>${props.cs_count.toLocaleString()}</span></td>
+          <td><span class='float-end'>${props.planned_cs_kw.toLocaleString()}kW</span></td>
+          <td><span class='float-end'>${props.planned_cs_count.toLocaleString()}</span></td>
         </tr>
         <tr>
           <td>Large DG</td>
           <td><span class='float-end'>${props.dg_large_kw.toLocaleString()}kW</span></td>
           <td><span class='float-end'>${props.dg_large_count.toLocaleString()}</span></td>
+          <td><span class='float-end'>${props.planned_dg_large_kw.toLocaleString()}kW</span></td>
+          <td><span class='float-end'>${props.planned_dg_large_count.toLocaleString()}</span></td>
         </tr>
         <tr>
           <td>Small DG</td>
           <td><span class='float-end'>${props.dg_small_kw.toLocaleString()}kW</span></td>
           <td><span class='float-end'>${props.dg_small_count.toLocaleString()}</span></td>
+          <td><span class='float-end'>${props.planned_dg_small_kw.toLocaleString()}kW</span></td>
+          <td><span class='float-end'>${props.planned_dg_small_count.toLocaleString()}</span></td>
         </tr>
       </tbody>
     </table>
   `
 }
 
-function updateLegend(layerSource, category){
-  let legendText = `<strong>${friendly_category_names[category]} kW of solar installed<br />by ${friendly_geography_names[layerSource]}</strong>`
+function updateLegend(layerSource, category, status){
```

Comment:
Oh no, jQuery 😭

Reply:
i'm using it for a few plugins I need:
 
 - jquery csv for converting a csv into a list of javascript objects
 - jquery address for updating and loading in query string parameters